src/app.js

```diff
@@ -595,7 +601,7 @@ function run () {
       this.fullLineMarker = null
       this.source = null
       if (lineColumnPos) {
-        this.source = compiler.lastCompilationResult.data.sourceList[location.file] // auto switch to that tab
+        this.source = compiler.getSourceName(location.file)
```

Comment:
Seems like a refactor too.

Reply:
yes sure but that line needs to be changed anyway cause `sourceList` is not here anymore. so better using `getSourceName` already